app/policies/application_policy.rb

```diff
@@ -7,4 +7,10 @@ class ApplicationPolicy < ActionPolicy::Base
   def allow_admins
     allow! if Current.groups.include?(Settings.authorization_workgroup_names.administrators)
   end
+
+  private
```

Comment:
private or protected?

Reply:
In my experience, `protected` is seldom (never?) a good idea in Ruby code.